sql/src/test/java/io/crate/integrationtests/WindowFunctionsIntegrationTest.java

```diff
@@ -195,4 +195,22 @@ public void testLimitAndOffsetIsAppliedCorrectlyWith2DifferentWindowDefinitions(
                "50| 3.0| 40.0\n")
         );
     }
+
+    @Test
+    public void testCollectSetUnboundedFollowingWindowFrame() {
```

Comment:
Isn't this fix completely unrelated to the windowing function / frame? Would probably be more isolated to use `collect_set` as a global aggregate function?